app.vue

```diff
@@ -36,7 +36,7 @@ store.setEnableModerationPanelToTrue(currentPath)
 
 const useLocalAPI = useRuntimeConfig().public.NUXT_USE_LOCAL_API
 
-initializeGqlClient(useLocalAPI)
+initializeGqlClient(useLocalAPI as string | undefined)
```

Comment:
move this up to when useLocalAPI is instantiated

Reply:
best practice for types: 
Add `as` casts when you want to change the type to something different or declare a new variable. If the variable is supposed to be of that type, put it in that line instead so it's more visible that it was expected to be of that type.